### DISCUSSION TITLE
React Native example

### DISCUSSION THREAD

**Discussion by sandeep-eroad** [prisma/prisma#5546] [Q&A] [answered]
Is there an example of Prisma with React Native showing migrations?

___________________________________________________________________________

I get following error while using Prisma in React-Native app

```
Unable to resolve module `.prisma/client/index-browser` from `node_modules/@prisma/client/index-browser.js`: 

None of these files exist:
  * node_modules/@prisma/.prisma/client/index-browser(.native|.ios.js|.native.js|.js|.ios.json|.native.json|.json|.ios.ts|.native.ts|.ts|.ios.tsx|.native.tsx|.tsx)
  * node_modules/@prisma/.prisma/client/index-browser/index(.native|.ios.js|.native.js|.js|.ios.json|.native.json|.json|.ios.ts|.native.ts|.ts|.ios.tsx|.native.tsx|.tsx)

RCTFatal
__28-[RCTCxxBridge handleError:]_block_invoke
_dispatch_call_block_and_release
_dispatch_client_callout
_dispatch_main_queue_callback_4CF
__CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__
__CFRunLoopRun
CFRunLoopRunSpecific
GSEventRunModal
-[UIApplication _run]
UIApplicationMain
main
start
```

Can anyone help me resolve this issue?

**Comment by alicantorun** [comment]
I have the same issue. any update on this?

**Reply by jharrell** [reply, marked as answer]
Hi all, 

Unfortunately, react native is not currently supported. We have an issue [here](https://github.com/prisma/prisma/issues/5011) tracking this request. Please go there and give that a 👍 !

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
